lms/static/sass/course/_info.scss

```diff
@@ -48,7 +48,7 @@ div.info-wrapper {
     width: 100%;
     display: block;
 
-    h1 {
+    h1,h3 {
```

Comment:
They are scoped within ``div.info-wrapper`` so they shouldn't be picked up elsewhere.